### PR TITLE
Fixed builder extend function override, instead of _name, name was used

### PR DESCRIPTION
**Pull request by furqanaziz**
If I want to extend by overriding any of the builder function already registered, it was not extending because of an underscore was missing in there. Version updation of  inflection library was also included for dist/. I have commited dist/* also in case you want to re-build it again with this tiny fix.

### CODE REVIEW

File: dist/angular-restmod-bundle.js [hunk 1]
```diff
@@ -171,7 +172,7 @@ angular.module('platanus.inflector', [])
           camelize: function(_string, _constant) {
             if (typeof _string !== 'string') return _string;
             return _string.replace(/(?:^[-_\s]*|[-_\s]+)([A-Z\d])/gi, function (match, _first, _index) {
-              return (!_constant && _index === 0) ? _first : _first.toUpperCase();
+              return (!_constant && _index === 0) ? _first.toLowerCase() : _first.toUpperCase();
```
Comment (monkeyci) [inline dist/angular-restmod-bundle.js:175]: Line is too long.

**Comment by furqanaziz** [comment]
Travis is failing because of missing credentials of saucelabs.com. I cannot put credentials for this repo but I have tested it with my fork and it gone through successfully.

**Comment by blackjid** [comment]
hey @furqanaziz I've merged https://github.com/platanus/angular-restmod/pull/398 with updated sauce labs credentials... if you want to rebase your PR

**Comment by blackjid** [comment]
also, I don't think we should be building the `dist` in the PR, with just build the librar on the releases. That't will help to maintain the PR small and easy to read.

**Comment by utf4** [comment]
That is perfect. I am closing this pull request out and will make another clean pull request. Hope you will find chance very soon to review my changes.

**Pull request closed without merging**